pythonClient/profile.py

```diff
@@ -30,7 +30,7 @@ def GetAClientCredentialsToken():
 
     sublessAuthUrl = os.getenv("issuerUrl")
     if sublessAuthUrl == None:
-        sublessAuthUrl = 'https://subless-test.auth.us-east-1.amazoncognito.com'
+        sublessAuthUrl = 'https://login.subless.com'
```

Comment:
(non-blocking) What's the intent behind changing this from test to normal login?

Reply:
That uri no longer exists. All the rest of the URIs point to prod, so this one does too.